cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -244,3 +247,21 @@ Then("There is a shadow effect between the parent row and first subrow", () => {
       "rgba(0, 20, 29, 0.1) 0px 6px 4px -4px inset"
     );
 });
+
+When("I have a large viewport", () => {
+  cy.viewport(700, 345);
+});
+
+When("I have a small viewport", () => {
+  cy.viewport(700, 240);
+});
+
+When("pageSize is selected", () => {
```

Comment:
we are performing `cy.click()` so it is not `selected` but `is opened` imo

Reply:
Amended the step in the feature file and in the step def to make it more readable.